checks.d/network.py

```diff
@@ -491,3 +496,117 @@ def _parse_solaris_netstat(self, netstat_output):
             metrics_by_interface[iface] = metrics
 
         return metrics_by_interface
+
+    @staticmethod
+    def _linux_sockstat_get_conntrack_max():
+        proc_name = '/proc/sys/net/ipv4/ip_conntrack_max'
+        try:
+            proc = open(proc_name,'r')
+            try:
+                lines = proc.readline()
+            finally:
+                proc.close()
+            return int(lines)
+        except IOError:
+            return 0
+
+    @staticmethod
+    def _linux_sockstat_get_conntrack_count():
+        proc_name = '/proc/sys/net/ipv4/netfilter/ip_conntrack_count'
+        try:
+            proc = open(proc_name,'r')
+            try:
+                lines = proc.readline()
+            finally:
+                proc.close()
+            return int(lines)
+        except IOError:
+            return 0
+
+    @staticmethod
+    def _linux_sockstat_get_max_orphans(self):
```

Comment:
If you're using `self`, then `@staticmethod` won't work. Either remove the decorator, or use `@classmethod`